tests/Sabberworm/CSS/ParserTest.php

```diff
@@ -766,4 +766,22 @@ function testLonelyImport() {
 		$sExpected = "@import url(\"example.css\") only screen and (max-width: 600px);";
 		$this->assertSame($sExpected, $oDoc->render());
 	}
+
+    function testTurnUnitLenient() {
+        $sText = ".foo {transform: rotate(1turn);}\n.bar {transform: rotate(1turns);}";
+        $sExpected = ".foo {transform: rotate(1turn);}\n.bar {transform: rotate(1turn);}";
```

Comment:
Question: Rather than truncating the unit to the part that is invalid, should not the property just be removed entirely?

For example, If I try using the `turns` unit, it is rejected entirely in Chrome:

![image](https://user-images.githubusercontent.com/134745/81601788-4fac2480-9380-11ea-8032-e47a540ad77c.png)

So I think this should actually rather be:

```suggestion
        $sExpected = ".foo {}\n.bar {}";
```

Reply:
Ah right, that makes sense. I've addressed this in https://github.com/sabberworm/PHP-CSS-Parser/pull/193/commits/54ae4fe617dafc6771db717cf85d986d4300275a.